requirements.txt

```diff
@@ -14,3 +14,4 @@ websocket-client==1.3.1
 netifaces==0.11.0; sys.platform == "win32"
 WMI==1.5.1; sys.platform == "win32"
 ./granulate-utils/
+./tests
```

Comment:
What is that for?

Reply:
assert_cast. I removed it it and added the implementation of assert_cast to gprofiler/metadata/system_metadata.py

Reply:
Ah I see, if needed then ofc move it to `gprofiler/utils` instead of installing the tests package :)